### ISSUE TITLE
Redirect correction

### ISSUE THREAD

**Issue by ReneeRendon** [rackerlabs/rackspace-how-to#1518]
This redirect needs to be investigated and corrected if necessary.

Redirect /how-to/general-purpose-cloud-servers-how-to-articles-and-other-resources/ URL to /how-to/cloud-servers-how-to-articles-other-resources

https://github.com/rackerlabs/nexus-control/blob/master/config/rewrites.d/support.rackspace.com.json

**Comment by catlook** [comment]
Ok this article exists: https://support.rackspace.com/how-to/cloud-servers-how-to-articles-other-resources/

So, the question is, did the other ever exist?  If it didn't, no redirect is needed.  If it did, we should leave the redirect.

**Comment by catlook** [comment]
And, drum-roll, it did.  I set the following alias in my profile: ff=git log --pretty=format: --name-status --all -M -B | sort -u | grep $1
Then, in my H2 repo directory I ran this command: ff general-purpose  to find any articles with general-purpose in the name.
Among the responses was: content/cloud-servers/general-purpose-cloud-servers-how-to-articles-and-other-resources.md

Therefore, we should keep the redirect, in case someone bookmarked the original article.
